.devcontainer/Dockerfile

```diff
@@ -13,9 +13,11 @@ RUN if [ "${NODE_VERSION}" != "none" ]; then su vscode -c "umask 0002 && . /usr/
 # RUN pip3 --disable-pip-version-check --no-cache-dir install -r /tmp/pip-tmp/requirements.txt \
 #    && rm -rf /tmp/pip-tmp
 
+RUN pip3 install --upgrade pip && pip3 install pip-tools
```

Comment:
Vi har det här redan:

```
	// Use 'postCreateCommand' to run commands after the container is created.
	"postCreateCommand": "make vscode_devcontainer_happy",
```

så lägg in det där så att det körs av Makefile istället. pip-tools har jag lagt in i requirements/test_requirements.in men det kanske inte har landat i main ännu.